src/main/webapp/confirmation.js

```diff
@@ -12,86 +12,95 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/** Gets a list of the user's scheduled sessions from the server and displays them on the page. */
-function getScheduledSessions() {
-    return getScheduledSessionsHelper(window);
-}
-
-//Helper function for getScheduledSessions, used for testing
-async function getScheduledSessionsHelper(window) {
-    await fetch('/confirmation', {method: 'GET'}).then((response) => {
+/** Creates a calendar with the Charts API and renders it on the page  */
+function createCalendar() {
+    fetch('/confirmation', {method: 'GET'}).then((response) => {
```

Comment:
Shouldn't the await still be here?

Reply:
It seemed to be working without it, but I added it back just in case.